docs/api_examples/template_path/make_html.py

```diff
@@ -1,5 +1,7 @@
 """
 this script builds html files with either classic or classic_clone templates
+
+Note: nbconvert 6.1 changed ``template_path`` to ``template_paths``
```

Comment:
Technically 6.0 changed it, but this is fine

Reply:
You know that I'm going to go back and change this ;-)

Reply:
Well I didn't want to get in the way of improvements merging :)